CCDB/include/CCDB/CcdbApi.h

```diff
@@ -545,6 +556,8 @@ class CcdbApi //: public DatabaseInterface
   mutable TGrid* mAlienInstance = nullptr;                       // a cached connection to TGrid (needed for Alien locations)
   bool mNeedAlienToken = true;                                   // On EPN and FLP we use a local cache and don't need the alien token
   static std::unique_ptr<TJAlienCredentials> mJAlienCredentials; // access JAliEn credentials
+  int mCurlRetries = 1;
```

Comment:
yes, if that's ok with everyone ? I wanted to have it disabled at first but if you think that it can be useful immediately then 3 seems like a good number.